### PR TITLE
LDT-725: User Profile Block

### PR DESCRIPTION
**Pull request by axelerant-hardik**

### CODE REVIEW

File: web/modules/custom/user_profile_block/user_profile_block.info.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: "User Profile Block"
+description: "Adds a block to manage user details"
+type: module
+package: Custom
+core_version_requirement: ^10
+
```
Comment (axelakshay) [inline web/modules/custom/user_profile_block/user_profile_block.info.yml:6]: extra space

File: web/modules/custom/user_profile_block/user_profile_block.info.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+dependencies:
+  - drupal:block
```
Comment (axelakshay) [inline web/modules/custom/user_profile_block/user_profile_block.info.yml:8]: add user dependency

File: web/modules/custom/user_profile_block/src/Plugin/Block/UserDetails.php [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Session\AccountProxyInterface $account
+   *   The current user service.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_manager
+   *   The entity type manager service.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, AccountProxyInterface $account, EntityTypeManagerInterface $entity_manager) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+
```
Comment (axelakshay) [inline web/modules/custom/user_profile_block/src/Plugin/Block/UserDetails.php:55]: extra space

File: web/modules/custom/user_profile_block/user_profile_block.info.yml (hunk 1 above)
Comment (axelerant-hardik) [reply web/modules/custom/user_profile_block/user_profile_block.info.yml:6]: This is just separating out things to look more clear similar to https://www.drupal.org/docs/develop/creating-modules/let-drupal-know-about-your-module-with-an-infoyml-file

File: web/modules/custom/user_profile_block/user_profile_block.info.yml (hunk 2 above)
Comment (axelerant-hardik) [reply web/modules/custom/user_profile_block/user_profile_block.info.yml:8]: Thanks, Done!

File: web/modules/custom/user_profile_block/src/Plugin/Block/UserDetails.php (hunk 3 above)
Comment (axelerant-hardik) [reply web/modules/custom/user_profile_block/src/Plugin/Block/UserDetails.php:55]: Isn't that just a normal line break? Separating the code with lines to make it look more clear. Is there anything else you are pointing to here?

Comment (axelakshay) [reply web/modules/custom/user_profile_block/src/Plugin/Block/UserDetails.php:55]: Yes that's my point and I have always taken a reference of core and contrib module, never seen such. So raised it, no issues then.

**Pull request closed without merging**